OpenOversight/app/db_repository/versions/006_migration.py

```diff
@@ -0,0 +1,42 @@
+from sqlalchemy import *
+from migrate import *
+
+
+from migrate.changeset import schema
+pre_meta = MetaData()
+post_meta = MetaData()
+unit_types = Table('unit_types', post_meta,
```

Comment:
what exactly is this table holding? units or unit types? unit type to me implies a general type with a one-to-many relationship from unit type to department but we make department id an int column, implying 1-to-1.

Reply:
also, ‘descrip’ feels like a weird choice of column name; is there a reason to not just use name here?
(asking these questions early while refactoring/migrations are still relatively easy)

Reply:
hmm so this table holds the unit names / descriptions, I linked them to department as I think it's nice to store it this way because when a user adds an officer assignment, the webapp could show them only the units that are associated with that police department